core/store/src/columns.rs

```diff
@@ -28,7 +28,7 @@ pub enum DBCol {
     /// - *Content type*: cell specific.
     BlockMisc,
     /// Column that stores Block content.
-    /// - *Rows*: block hash (CryptHash)
```

Comment:
It's strange that it wasn't caught by cspell. Can you check if this wasn't added by accident to any of the cspell.json files please? If so please remove it from there.

Reply:
I don't see it in any of those files, is cspell expected to be enforced in CI?

Reply:
yes, it's called spellcheck and I see it was ran on your PR. Maybe it's incremental? Anyway, don't worry about it.